spec-tests/src/test/java/org/candlepin/spec/jobs/JobStatusSpecTest.java

```diff
@@ -0,0 +1,480 @@
+    public void shouldAllowPagingOfJobs() throws Exception {
+        AsyncJobStatusDTO job1 = ownerApi.healEntire(owner.getKey());
+        AsyncJobStatusDTO job2 = ownerApi.healEntire(owner.getKey());
+        AsyncJobStatusDTO job3 = ownerApi.healEntire(owner.getKey());
+        List<AsyncJobStatusDTO> jobs = jobsClient.listJobStatuses(null, null, null,
+            Set.of(owner.getKey()), null, null, null, null, null, 2, 1, null, null);
+        // Since we're not setting the order, we can't guarantee which job we'll get, just that we'll
+        // get exactly one of them.
+        assertEquals(1, jobs.size());
+        List<String> ids = List.of(job1.getId(), job2.getId(), job3.getId());
+        List<AsyncJobStatusDTO> result = jobs.stream()
+            .filter(x -> ids.contains(x.getId()))
+            .collect(Collectors.toList());
+        assertEquals(1, result.size());
+    }
+
+    @Test
+    @DisplayName("should allow sorting of jobs")
+    public void shouldAllowSortingOfJobs() throws Exception {
+        AsyncJobStatusDTO job1 = ownerApi.healEntire(owner.getKey());
+        AsyncJobStatusDTO job2 = ownerApi.healEntire(owner.getKey());
+        AsyncJobStatusDTO job3 = ownerApi.healEntire(owner.getKey());
+        List<AsyncJobStatusDTO> jobs = jobsClient.listJobStatuses(null, null, null,
+            Set.of(owner.getKey()), null, null, null, null, null, null, null, "asc", "id");
+        // Since we're not setting the order, we can't guarantee which job we'll get, just that we'll
+        // get exactly one of them.
+        assertEquals(3, jobs.size());
+        // Verify that the IDs are all in ascending order
+        String lastId = null;
+        for (AsyncJobStatusDTO job : jobs) {
+            if (lastId != null) {
+                assertThat(lastId.compareTo(job.getId())).isLessThan(0);
+            }
+            lastId = job.getId();
+        }
+    }
+
+    @Test
+    @DisplayName("should allow paging and sorting of jobs")
+    public void shouldAllowPagingAndSortingOfJobs() throws Exception {
+        AsyncJobStatusDTO job1 = ownerApi.healEntire(owner.getKey());
+        AsyncJobStatusDTO job2 = ownerApi.healEntire(owner.getKey());
+        AsyncJobStatusDTO job3 = ownerApi.healEntire(owner.getKey());
+        List<AsyncJobStatusDTO> jobs = jobsClient.listJobStatuses(null, null, null,
+            Set.of(owner.getKey()), null, null, null, null, null, 3, 1, "desc", "id");
+        assertEquals(1, jobs.size());
+        assertEquals(job1.getId(), jobs.get(0).getId());
+    }
+
+    @Test
+    @DisplayName("should allow user to view job status of consumer in managed org")
+    public void shouldAllowUserToViewConsumerJobStatus() throws Exception {
+        ConsumerDTO consumer = new Consumers.Builder()
+            .withOwner(owner)
+            .build();
+        consumer = consumerApi.createConsumer(consumer, user.getUsername(), owner.getKey(), null, false);
+        // need a consumer api client
```

Comment:
Should this be removed? The only reason why I don't think that we need the comment is that the variable name describes that it is a consumer api client. This also applies to the other methods that have the same comment.